lib/rex/post/meterpreter/ui/console/command_dispatcher/core.rb

```diff
@@ -816,16 +816,18 @@ def cmd_transport(*args)
   end
 
   @@migrate_opts = Rex::Parser::Arguments.new(
-    '-p'  => [true,  'Writable path - Linux only (eg. /tmp).'],
-    '-t'  => [true,  'The number of seconds to wait for migration to finish (default: 60).'],
-    '-h'  => [false, 'Help menu.']
+    '-P' => [true, 'PID to migrate to.'],
```

Comment:
Ah good stuff. Thanks mate :+1: I should read further and not make assumptions :)